presto-main/src/test/java/com/facebook/presto/cost/TestSemiJoinStatsCalculator.java

```diff
@@ -204,7 +204,7 @@ public void testAntiJoin()
                 .outputRowsCount(inputStatistics.getOutputRowCount() * xStats.getValuesFraction() * 0.5);
 
         // source stats are unknown
-        assertThat(computeSemiJoin(inputStatistics, inputStatistics, unknown, u))
+        assertThat(computeAntiJoin(inputStatistics, inputStatistics, unknown, u))
```

Comment:
whu did you change this from compute semiJoin to computeAntiJoin?  That's not a misspelling that's a different thing

Reply:
I think that was a misspelling. `computeSemiJoin` in the other test method within the same class.

Reply:
Yeah, too much for a missspelling. Let's call it a mistake =)